cogs/ama.py

```diff
@@ -1,138 +1,136 @@
+import discord
 from discord.ext import commands
-from discord.ext.commands import group
 from datetime import datetime
 from cogs.base_cog import BaseCog
 
 
-class Ama(BaseCog):
-    def __init__(self, bot):
-        super().__init__(bot)
-
-    @group()
+class Ama(BaseCog, name="Ask me anything"):
+    @commands.group(invoke_without_command=True)
     async def qna(self, ctx):
         """Ask me Anything commands."""
-        bot_channel = self.bot.get_channel(332644650462478336)
-        if bot_channel is None:
-            await ctx.channel.send('The dedicated bot commands channel cannot be found')
-            return False
-        if ctx.invoked_subcommand is None:
-            await bot_channel.send('Invalid Ask me Anything command passed, {}'.format(ctx.author.mention))
-
-    @qna.command(description='')
+        await ctx.invoke(self.bot.get_command("help"), ctx.invoked_with)
+
+    @qna.command()
     @commands.guild_only()
     @commands.has_any_role('Nixie', 'Supervisors', 'Moderators')
```

Comment:
This can be improved later on but .has_any_role should be either config or DB driven.  hard coded is a bad pattern

Reply:
yea, im working on this locally